service/db/src/main/java/gov/va/vro/service/db/SaveToDbServiceImpl.java

```diff
@@ -152,8 +156,8 @@ public void insertOrUpdateExamOrderingStatus(ExamOrder examOrder) {
             .findByCollectionId(examOrder.getCollectionId())
             .orElseGet(() -> createExamOrder(examOrder));
     if (null != examOrderEntity) {
-      examOrderEntity.setCollectionId(examOrder.getCollectionId());
```

Comment:
Since we just looked up the examOrderEntity by this key, and keys should not change, i removed this. It technically would be equal and not corrupt data but I felt it better to ditch it.